graphics/babl/Portfile

```diff
@@ -60,6 +59,10 @@ if {[variant_isset universal]} {
     destroot.env-append    "CC=${configure.cc} ${configure.cc_archflags}"
 }
 
+variant doc description {Generate optional docs} {
```

Comment:
Please use "docs" for the variant name. See https://trac.macports.org/ticket/58338 for more information.

Reply:
Done in https://github.com/macports/macports-ports/pull/9176/commits/e9f82d6d05b2ed285e776e5c6d010f47e19aa52d